testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDatabricksCompatibilityCleanUp.java

```diff
@@ -37,52 +36,45 @@
 import static java.lang.String.format;
 import static java.lang.System.currentTimeMillis;
 import static java.util.Objects.requireNonNull;
-import static java.util.concurrent.TimeUnit.DAYS;
 import static java.util.concurrent.TimeUnit.MINUTES;
 import static java.util.stream.Collectors.toUnmodifiableList;
 
 public class TestDatabricksCompatibilityCleanUp
         extends ProductTest
 {
     private static final Logger log = Logger.get(TestDatabricksCompatibilityCleanUp.class);
-    private static final long SCHEMA_CLEANUP_THRESHOLD_SECONDS = (currentTimeMillis() / 1000) - DAYS.toSeconds(7);
+    private static final Instant SCHEMA_CLEANUP_THRESHOLD = Instant.now().minus(7, ChronoUnit.DAYS);
     private static final long MAX_JOB_TIME_MILLIS = MINUTES.toMillis(5);
 
     @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
     public void testCleanUpOldTablesUsingDelta()
-            throws Exception
     {
-        String hiveMetastoreUri = requireNonNull(System.getenv("HIVE_METASTORE_URI"), "Environment HIVE_METASTORE_URI was not set");
-        URI metastoreUri = URI.create(hiveMetastoreUri);
+        AWSGlueAsyncClientBuilder asyncGlueClientBuilder = AWSGlueAsyncClientBuilder.standard();
+        String awsRegion = requireNonNull(System.getenv("AWS_REGION"), "Environment AWS_REGION was not set");
+        asyncGlueClientBuilder.setRegion(awsRegion);
```

Comment:
I don't think you need to do these two lines. The region should pull from the environment variable automatically

Reply:
You're right. Thank you for hinting this.